src/monitor-specs.js

```diff
@@ -1,36 +1,262 @@
+    throw new Error(`GraphQL error, could not add "${label}" label to issue #${session.number}`);
+  }
 }
-const review_list = review_needed.map(c => `- [ ] ${c.url} updated on ${c.lastupdated}; last comment: “${c.comment}” made on ${c.lastreviewed}`).join("\n");
-core.exportVariable("review_list", review_list);
-console.log(review_list);
-
-// Update monitor.json setting lastreviewed date today on all entries
-// This will serve as input to the automated pull request
-Object.values(monitorList.repos).forEach(r => r.lastreviewed = today);
-Object.values(monitorList.specs).forEach(r => r.lastreviewed = today);
-await fs.writeFile("./src/data/monitor.json", JSON.stringify(monitorList, null, 2));
+
+
+/**
+ * Add the "review" label to the given issue, along with a comment
+ */
+let reviewLabelId = null;
+async function flagIssueForReview(issue, comment) {
+  if (comment) {
+    // Using a variable to avoid having to deal with comment escaping issues
+    const commentResponse = await sendGraphQLQuery(`
+      mutation($comment: AddCommentInput!) {
+        addComment(input: $comment) {
+          subject {
+            id
+          }
+        }
+      }`, {
+        comment: {
+          subjectId: issue.id,
+          body: comment,
+          clientMutationId: "mutatis mutandis"
+        }
+      },
+      githubToken);
+    if (!commentResponse?.data?.addComment?.subject?.id) {
+      console.log(JSON.stringify(commentResponse, null, 2));
+      throw new Error(`GraphQL error, could not add comment to issue #${issue.number}`);
+    }
+  }
+
+  await setIssueLabel(issue, "review");
+}
+
+
+fetchIssuesToReview().then(async issues => {
+  const issuesToReview = [];
+  for (const issue of issues) {
+    const lastReviewedEvent = issue.timelineItems.nodes.find(event =>
+      event.label.name === "review");
+    issue.lastReviewed = (new Date(lastReviewedEvent ?
+        lastReviewedEvent.createdAt :
+        issue.createdAt))
+      .toJSON()
+      .slice(0, 10);
+
+    const sections = splitIssueBodyIntoSections(issue.body);
+    const urlSection = sections.find(section => section.title === 'URL');
+    if (!urlSection) {
+      console.warn(`- ${issue.title} (#${issue.number}) does not follow the expected issue format`);
+      if (updateGitHubIssues) {
+        await setIssueLabel(issue, "invalid");
+      }
+      continue;
+    }
+
+    // Retrieve the spec and check the last-modified HTTP header
+    const response = await fetch(urlSection.value);
+    const { headers } = response;
+
+    // The CSS drafts use a proprietary header to expose the real last
+    // modification date
+    issue.lastRevised = (new Date(headers.get('Last-Revised') ?
+        headers.get('Last-Revised') :
+        headers.get('Last-Modified')))
+      .toJSON()
+      .slice(0, 10);
+    if (issue.lastRevised > issue.lastReviewed) {
+      issuesToReview.push(issue);
+    }
+    // We don't need the response's body, but not reading it means Node will keep
```

Comment:
for a separate PR, but I noticed that entries in https://chromestatus.com/features.json have a `updated.when` property we could also use to update monitored specs

Reply:
Filed #1765